EditorExtensions/Resources/server/services/srv-scss.js

```diff
@@ -35,8 +35,9 @@ var handleSass = function(writer, params) {
             writer.end();
         }
 
-        var css = result.css.toString();
-        var map = result.map.toString();
+        var map, css = result.css.toString();
+        if (!options.omitSourceMapUrl)
+            map = result.map.toString();
```

Comment:
`omitSourceMapUrl` option in node-sass does not indicate the presence of source-map, but only the URL. 
In Web Essentials, we need source-maps in all cases for editor enhancements.
I didn't think of autoprefixer use case before, where it would not be able to update the map file accordingly (since `sourceMappingURL` acts as a bridge). For that matter, we would need to inject the `sourceMappingURL` here to CSS:

```javascript
var css = result.css.toString();  
var map = result.map.toString(); 
var appendix;

if (!options.omitSourceMapUrl) {
    appendix = '\n/*# sourceMappingURL= ' + params.mapFileName + ' */';
    css += appendix;
}
```

Then later on:

```diff
- css = autoprefixedOutput.css;
+ css = autoprefixedOutput.css.replace(appendix, '');

- RtlContent: rtlResult.css,
+ RtlContent: rtlResult.css.replace(appendix, ''),
```